lib/services/dgg-rolling-chat-cache.js

```diff
@@ -62,6 +77,28 @@ class ChatCache {
     this.tombStoneMap[user] = moment().unix();
   }
 
+  addMessageToViewerUrlMap(user, message) {
+    if (
+      this.messageUrlSpamUser &&
+      !this.messageMatching.mentionsUser(message, this.messageUrlSpamUser)
+    ) {
+      return;
+    }
+    this.messageMatching.getLinks(message).forEach((link) => {
+      if (!_.has(this.viewerUrlMap, user)) this.viewerUrlMap[user] = [];
+      this.viewerUrlMap[user].push(`${moment().unix()}%${link.hostname + link.pathname}`);
```

Comment:
Is there a compelling reason not to represent this data as an object? The %-delimited string seems to just make things more complex with the constant parsing and splitting.